test/test_images/performance/dev.config

```diff
@@ -0,0 +1,58 @@
+### Creating this benchmark:
```

Comment:
This file can be removed, as we have other ways to pass the config to the running benchmark.

Reply:
Are you sure? I thought it's needed to run the aggregator and to run tests in CI

Reply:
Yes, I'm pretty sure it can be removed.
Currently we are passing benchmark key and name as args for the continuous run jobs, see https://github.com/knative/eventing/blob/8e38791d3e4cba16df132c0b42ba68f62a746eac/test/performance/benchmarks/broker-imc/continuous/200-broker-imc.yaml#L103.

After your PR in pkg is merged, I can switch to also using configmap in a following PR.

Reply:
https://github.com/knative/eventing/pull/2202